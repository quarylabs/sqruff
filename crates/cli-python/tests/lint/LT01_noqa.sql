SELECT   1; --noqa: LT01
