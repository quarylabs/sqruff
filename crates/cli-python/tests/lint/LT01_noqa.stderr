The linter processed 1 file(s).
All Finished
